common/baselines/tedlium2_v2/data.py

```diff
@@ -0,0 +1,42 @@
+from collections import defaultdict
+from typing import Dict
+
+from i6_experiments.common.datasets.tedlium2.constants import CONCURRENT
```

Comment:
While I know this does not change, if we do a "full copy" shouldn't we for completeness also copy this? @JackTemaki opinions?